### PR TITLE
13.0 mig account fiscal year

### PR DESCRIPTION
**Pull request by jimhoefnagels**

### CODE REVIEW

**Review by eLBati — CHANGES_REQUESTED** [review]
Please add `oca_dependencies.txt` file like this one https://github.com/OCA/account-financial-tools/blob/12.0/oca_dependencies.txt

File: account_fiscal_year/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+This module just adds the menu
+
+Invoicing > Configuration > Accounting > Fiscal Years
+
+This is totally refactored compared to 11 version, because odoo 12 introduced `account.fiscal.year` object.
+See https://github.com/OCA/account-financial-tools/pull/706
```
Comment (eLBati) [inline account_fiscal_year/readme/DESCRIPTION.rst]: I think we can remove these 2 lines now.

Then, OK

**Comment by eLBati** [comment]
@jimhoefnagels could you squash the last 2 commits in 1?

**Review by eLBati — APPROVED** [review]
Thanks

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]
Already done in https://github.com/OCA/account-financial-tools/pull/900.

**Comment by jimhoefnagels** [comment]
> Already done in #900.

Could you mention this in https://github.com/OCA/account-financial-tools/issues/888

**Comment by eLBati** [comment]
> Could you mention this in #888

Done.

@jimhoefnagels so this can be closed if you agree

**Pull request closed without merging**